app/views/templates/ssh/puppet_run_once.erb

```diff
@@ -3,7 +3,7 @@ kind: job_template
 name: Puppet Run Once - SSH Default
 model: JobTemplate
 job_category: Puppet
-description_format: 'Run Puppet once with "%{puppet_options}"'
+description_format: 'Run Puppet once'
```

Comment:
is this still necessary? also note that changes to job templates are supposed to go through [community-templates](https://github.com/theforeman/community-templates/) repo

Reply:
yes, it's not necessary, thanks